### PR TITLE
Portuguese Brazilian Localization

### PR DESCRIPTION
**Pull request by Alstin112**

### CODE REVIEW

**Review by UltimatChamp — CHANGES_REQUESTED** [review]
@Alstin112 I've recently added some new options.
Could you please translate them as well? I've included them in your Pull Request repo.

File: src/main/resources/assets/bettergrass/lang/pt_br.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "bettergrass.title": "Configuração BetterGrassify",
+  "bettergrass.betterGrassMode": "Modo de Melhor Grama",
+  "bettergrass.betterGrassMode.desc": "§4Desligado §r- Blocos de grama aparentam normal\n§3Rápido §r- A textura do topo da grama será aplicado em todos os lados do bloco (rápido)\n§dBonito §r- A textura do topo de grama será aplicado dinamicamente aos lados do bloco (lento)",
```
Comment (UltimatChamp) [inline src/main/resources/assets/bettergrass/lang/pt_br.json:4]:
```suggestion
  "bettergrass.betterGrassMode.desc": "§4Não §r- Blocos de grama aparentam normal\n§3Rápidas §r- A textura do topo da grama será aplicado em todos os lados do bloco (rápido)\n§dDetalhadas §r- A textura do topo de grama será aplicado dinamicamente aos lados do bloco (lento)",
```

File: src/main/resources/assets/bettergrass/lang/pt_br.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "bettergrass.snowy.desc": "Conecta grama com neve também!",
+  "bettergrass.betterSnowMode": "Better Snow",
```
Comment (Alstin112) [inline src/main/resources/assets/bettergrass/lang/pt_br.json:6]:
```suggestion
  "bettergrass.betterSnowMode": "Neve Melhorada",
```

File: src/main/resources/assets/bettergrass/lang/pt_br.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "bettergrass.betterSnowMode.desc": "§4Off §r- No snow layer will be generated.\n§3OptiFine-like §r- Shows a snow layer beneath specific blocks that have at-least one snow layer or snow block near them.\n§dLambdaBetterGrass-like §r- Shows a snow layer beneath specific blocks that have at-least \"two\" snow layers or snow blocks near them.",
+  "bettergrass.betterSnowMode.optifine": "OptiFine-like",
```
Comment (Alstin112) [inline src/main/resources/assets/bettergrass/lang/pt_br.json:8]:
```suggestion
  "bettergrass.betterSnowMode.optifine": "Tipo-OptiFine",
```

File: src/main/resources/assets/bettergrass/lang/pt_br.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "bettergrass.betterSnowMode.lambda": "LambdaBetterGrass-like",
```
Comment (Alstin112) [inline src/main/resources/assets/bettergrass/lang/pt_br.json:9]:
```suggestion
  "bettergrass.betterSnowMode.lambda": "Tipo-LambdaBetterGrass",
```

File: src/main/resources/assets/bettergrass/lang/pt_br.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  "bettergrass.grassBlocks.desc": "Conecta bloco de grama também!",
+  "bettergrass.dirtPaths.desc": "Conecta caminhos de grama também!\n\n§cTalvez cause alguns bugs visuais!",
```
Comment (Alstin112) [inline src/main/resources/assets/bettergrass/lang/pt_br.json:11]:
```suggestion
  "bettergrass.dirtPaths.desc": "Conecta caminho de terra também!\n\n§cTalvez cause alguns bugs visuais!",
```

File: src/main/resources/assets/bettergrass/lang/pt_br.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+  "bettergrass.title": "Configuração BetterGrassify",
+  "bettergrass.betterGrassMode": "Modo de Melhor Grama",
```
Comment (Alstin112) [inline src/main/resources/assets/bettergrass/lang/pt_br.json:3]:
```suggestion
  "bettergrass.betterGrassMode": "Modo de Grama Melhorada",
```

**Review by Alstin112 — COMMENTED** [review]
Translated the new text

**Review by UltimatChamp — APPROVED** [review]

**Pull request merged**